### ISSUE TITLE
Open in new tab,not working, Cmd/Ctrl + Return

### ISSUE THREAD

**Issue by jukrb0x** [alyssaxuu/omni#79]
Hi, thanks for the amazing extension, as checked the prev issue #13 that confirmed the open in the tab functionality. But I've tried it on both mac (Cmd) and windows(Ctrl) + Return and it still opened the link in current window, wish this could be fixed xd.

**Comment by alyssaxuu** [comment]
That's odd. What browser are you using? Are you accessing bookmarks or another type of action? Both cmd + click and cmd + return should work to open in new tab.

**Comment by jukrb0x** [comment]
Currently, I am using Chrome 97.0.4692.71 (Official Build) (x86_64) on macOS. Let me show you with keystrokes recording.

I've tested both tries.

https://user-images.githubusercontent.com/15688641/157700034-eaa283af-8267-4fca-b6cb-84882ece840a.mov

**Comment by jukrb0x** [comment]
it doesn't work both on mac and win, so I guess it could relate to some extensions running in the chrome, will do some ablation experiment later on.

**Comment by FatehAK** [comment]
Yes can confirm, the new tab shortcut doesn't seem to work. I tried it on an incognito window as well.

OS: Mac OS Monterey
Chrome Version 106.0.5249.119 (Official Build) (arm64)
